lib/ansible/modules/clustering/oc.py

```diff
@@ -0,0 +1,478 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# (c) 2017, Kenneth D. Evensen <kevensen@redhat.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
```

Comment:
Could you please change from Line 4 to 18, to https://gist.github.com/abadger/35f6918ac1ce48f97cc173c35a718d76#file-header-boilerplate-txt